SKPhotoBrowser/SKZoomingScrollView.swift

```diff
@@ -225,7 +225,9 @@ public class SKZoomingScrollView:UIScrollView, UIScrollViewDelegate, SKDetecting
     }
     
     func handleDoubleTap(view: UIView, touch: UITouch) {
-        // nothing to do
+        var point = self.convertPoint(touch.locationInView(view), toView: self.photoImageView)
+        point.y = 0
```

Comment:
If not set `point.y = 0`, the `point.y` will be `< 0`, is a negative number.
Maybe set point's y to 0 is better than origin `point.y`